tests/template_tests/test_context.py

```diff
@@ -158,6 +158,24 @@ def test_flatten_context_with_context(self):
             },
         )
 
+    def test_flatten_context_with_context_copy(self):
+        """
+        #35417 -- flatten() on Context.new() contexts shouldn't fail.
+        """
```

Comment:
This docstring is unnecessary, please remove it. 
```suggestion
        """
        #35417 -- flatten() on Context.new() contexts shouldn't fail.
        """
```
```suggestion
```